types/case-sensitive-paths-webpack-plugin/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/Urthen/case-sensitive-paths-webpack-plugin#readme
 // Definitions by: Andrew Makarov <https://github.com/r3nya>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
Why the change to TS 2.2?

Reply:
Because of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23578/files#diff-f71dc9c7400483f83b0612cb695193d4R80